landing-page/index.html

```diff
@@ -0,0 +1,33 @@
+<html>
+  <head>
+    <style>
+      body {
+  background-color: MistyRose;
+}
+      </style>
+  </head>
+  <body>
+      <style>
+          h1{
+  background-color: white;
+  color: MistyRose;
+  font: 60pt Impact, Charcoal, sans-serif;
```

Comment:
The formatting of the values on the font property aren't quite right here, so this won't be properly applied. It should be something like `font: 60px "Impact", sans-serif` (see here for an example: https://developer.mozilla.org/en-US/docs/Web/CSS/font). Also, and you may not have covered this in class yet, this font is likely not to be applied. The font file either has to be included directly in your project, or you have to have a link to a Google font, for instance, in order to be applied on the page.